docs/deployment/content/remote-local-setup.yaml

```diff
@@ -0,0 +1,85 @@
+          set -ex
+          cd
+          apk add bash bash-completion curl fzf git jq less lsof make mandoc mc procps tmux tmux-doc vim
+          apk add --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main mount
+          echo golang            && curl -sLO "https://go.dev/dl/$(curl -sL https://golang.org/VERSION?m=text).linux-amd64.tar.gz" && tar -C /usr/local -xzf go1.*.linux-amd64.tar.gz
+          echo helm              && curl -sLO "https://get.helm.sh/helm-$(curl -sL https://api.github.com/repos/helm/helm/releases/latest | jq .tag_name -r)-linux-amd64.tar.gz" && tar -xzf helm-*-linux-amd64.tar.gz && mv linux-amd64/helm /usr/local/bin/helm
+          echo kind              && curl -sL  "https://kind.sigs.k8s.io/dl/$(curl -sL https://api.github.com/repos/kubernetes-sigs/kind/releases/latest | jq .tag_name -r)/kind-linux-amd64" -o /usr/local/bin/kind && chmod +x /usr/local/bin/kind
+          echo kns               && curl -sL  "https://raw.githubusercontent.com/blendle/kns/master/bin/kns" -o /usr/local/bin/kns && chmod +x /usr/local/bin/kns
+          echo ktx               && curl -sL  "https://raw.githubusercontent.com/blendle/kns/master/bin/ktx" -o /usr/local/bin/ktx && chmod +x /usr/local/bin/ktx
+          echo kube-ps1          && curl -sL  "https://raw.githubusercontent.com/jonmosco/kube-ps1/master/kube-ps1.sh" -o ~/.kube-ps1.sh
+          echo kubectl           && curl -sL  "https://dl.k8s.io/release/$(curl -sL https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" -o /usr/local/bin/kubectl && chmod +x /usr/local/bin/kubectl
+          echo tmux-completion   && curl -sL  "https://raw.githubusercontent.com/imomaliev/tmux-bash-completion/master/completions/tmux" -o /usr/share/bash-completion/completions/tmux
+          echo docker-completion && curl -sL  "https://raw.githubusercontent.com/docker/docker-ce/master/components/cli/contrib/completion/bash/docker" -o /usr/share/bash-completion/completions/docker
+          echo '127.0.0.1 api.local.local.external.local.gardener.cloud' >> /etc/hosts
+          echo 'source ~/.bashrc' > ~/.bash_profile
+          cat > ~/.bashrc <<"EOF"
+            export PATH=$PATH:/usr/local/go/bin
+            export KUBECONFIG=~/gardener/example/gardener-local/kind/kubeconfig:/tmp/kubeconfig-shoot-local.yaml
+            source <(kubectl completion bash)
+            alias k=kubectl
+            complete -o default -F __start_kubectl k
+            source ~/.kube-ps1.sh
+            export PS1='[\w $(printf "$(kube_ps1)")]\$ '
+            cd ~/gardener
+          EOF
+          cat > ~/.tmux.conf <<"EOF"
+            set -g mouse on
+            set -g mode-keys vi
+            set -g default-shell /bin/bash
+            set -g pane-border-status top
+            set -g pane-border-format " #{pane_index} #{pane_title} - #{pane_current_command} "
+          EOF
+          git clone -q https://github.com/gardener/gardener.git
+          dockerd-entrypoint.sh &
+          sleep 7
```

Comment:
Where is the magic `7` coming from? 😄

Reply:
That was my first attempt to wait for dockerd to start up. 😄

This could allow to actually execute some of the commands while preparing the tmux session in the next few lines. Note that the currently they are not executed, but only "prepared".

I removed the magic number: 😄

https://github.com/gardener/gardener/pull/6730/files#diff-2bce7f51d29fa6c60d9e9f4e764c4da6a1c2a697959e3e890402daeca6895594R56

```
until docker ps >/dev/null 2>&1; do sleep 1; done
```